oss-internship-2020/gdal/raster.cc

```diff
@@ -0,0 +1,144 @@
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+#include <syscall.h>
+
+#include <iostream>
+
+#include "gdal_sapi.sapi.h"
+#include "sandboxed_api/sandbox2/util/fileops.h"
+
+class GdalSapiSandbox : public gdalSandbox {
+ public:
+  std::string filePath;
```

Comment:
This member should be `private` instead. C++ style guide also mandates the name `file_path_`.

See https://google.github.io/styleguide/cppguide.html#Variable_Names